### PR TITLE
add: A futuristic personalized command line calculator

### PR DESCRIPTION
**Pull request by NitkarshChourasia**
With multiple program's features updates.
@PBJI

### CODE REVIEW

**Comment by NitkarshChourasia** [comment]
@PBJI

**Comment by NitkarshChourasia** [comment]
@PBJI

**Comment by NitkarshChourasia** [comment]
@PBJI Please have a look at it.
Thank you.
@NitkarshChourasia

**Review by NitkarshChourasia — COMMENTED** [review]
Awesome.

File: advanced_calculator.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+# Yes, I should be making it.
+__author__ = "Nitkarsh Chourasia"
+__version__ = "0.0.0"  # SemVer # Understand more about it
+__license__ = "MIT"  # Understand more about it
+# Want to make it open source but how to do it?
+# Program to make a simple calculator
+# Will have to extensively work on Jarvis and local_document and MongoDb and Redis and JavaScript and CSS and DOM manipulation to understand it.
+# Will have to study maths to understand it more better.
+# How can I market gtts? Like showing used google's api? This is how can I market it?
+# Project description? What will be the project description?
```
Comment (PBJI) [inline advanced_calculator.py:11]: Trying to integrate gtts and give accounting users to calculate things faster?

File: advanced_calculator.py [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+from numbers import Number
+from sys import exit
+import colorama as color
```
Comment (PBJI) [inline advanced_calculator.py:15]: Also adding terminal based controls

File: advanced_calculator.py [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+
+    def leap_year(self):
+        """_summary_: Get the leap year of the user"""
+        pass
+
+    def perimeter(self):
+        """_summary_: Get the perimeter of the user"""
+        pass
+
+    class Trigonometry:
```
[the remark this answers is not in the document]
Comment (NitkarshChourasia) [reply advanced_calculator.py:170]: Yes?
Doesn't inheritance and descendant things exists?

File: advanced_calculator.py (hunk 2 above)
Comment (NitkarshChourasia) [reply advanced_calculator.py:15]: Just to exit the program out of while loop which takes continuous inputs like a actual calculator and when power is pressed -> sys.exit() is executed, suppose.

File: advanced_calculator.py [hunk 4]
```diff
@@ -0,0 +1,391 @@
+# This is like making a package.lock file for npm package.
+# Yes, I should be making it.
+__author__ = "Nitkarsh Chourasia"
+__version__ = "0.0.0"  # SemVer # Understand more about it
+__license__ = "MIT"  # Understand more about it
+# Want to make it open source but how to do it?
+# Program to make a simple calculator
+# Will have to extensively work on Jarvis and local_document and MongoDb and Redis and JavaScript and CSS and DOM manipulation to understand it.
```
[the remark this answers is not in the document]
Comment (NitkarshChourasia) [reply advanced_calculator.py:8]: That was a future vision as of just now.

**Comment by NitkarshChourasia** [comment]
@PBJI also tell me more things which can be added like features, functions , _operation types especially_.
Thank You, for your time. @PBJI 
As always, @NitkarshChourasia ready to help you anytime.

**Comment by NitkarshChourasia** [comment]
@geekcomputers

**Pull request merged**